ts/store/reducers/backoffError.ts

```diff
@@ -2,67 +2,16 @@ import { fromNullable } from "fp-ts/lib/Option";
 import { Millisecond } from "italia-ts-commons/lib/units";
 import { getType } from "typesafe-actions";
 import _ from "lodash";
-import { PayloadAC } from "typesafe-actions/dist/type-helpers";
 import { index } from "fp-ts/lib/Array";
-import {
-  bpdTransactionsLoadPage,
-  bpdTransactionsLoadRequiredData
-} from "../../features/bonus/bpd/store/actions/transactions";
-import { mvlDetailsLoad } from "../../features/mvl/store/actions";
 import { Action } from "../actions/types";
-import {
-  fetchTransactionsFailure,
-  fetchTransactionsSuccess
-} from "../actions/wallet/transactions";
-import {
-  addWalletCreditCardFailure,
-  addWalletCreditCardSuccess,
-  fetchWalletsFailure,
-  fetchWalletsSuccess
-} from "../actions/wallet/wallets";
-import { bpdLoadActivationStatus } from "../../features/bonus/bpd/store/actions/details";
-import { bpdPeriodsAmountLoad } from "../../features/bonus/bpd/store/actions/periods";
-import { euCovidCertificateGet } from "../../features/euCovidCert/store/actions";
-import {
-  svPossibleVoucherStateGet,
-  svVoucherListGet
-} from "../../features/bonus/siciliaVola/store/actions/voucherList";
-import { svGetPdfVoucher } from "../../features/bonus/siciliaVola/store/actions/voucherGeneration";
 import { computedProp } from "../../types/utils";
 import { GlobalState } from "./types";
-
-/**
- * list of monitored actions
- * each entry is a tuple of 2
- * 0 - the failure action that is considered to create/increment the backoff delay
- * 1 - the success action that is considered to delete the previous backoff delay
- */
-const monitoredActions: ReadonlyArray<
```

Comment:
this is dead code I guess?

Reply:
Yep, nothing has actually changed in the code, I just extracted some internal functions for testing purposes